assets/src/block-validation/error/error-panel-title.js

```diff
@@ -38,9 +34,11 @@ export function ErrorPanelTitle( { blockType, title, error: { type }, status } )
 	return (
 		<>
 			<div className="amp-error__icons">
-				<div className={ `amp-error__error-type-icon amp-error__error-type-icon--${ type.replace( /_/g, '-' ) }` }>
-					<ErrorTypeIcon type={ type } />
-				</div>
+				{ type && (
+					<div className={ `amp-error__error-type-icon amp-error__error-type-icon--${ type?.replace( /_/g, '-' ) }` }>
```

Comment:
Optional chaining is unnecessary here since `type &&` ensures  it is defined, no?

```suggestion
					<div className={ `amp-error__error-type-icon amp-error__error-type-icon--${ type.replace( /_/g, '-' ) }` }>
```